build/package/Dockerfile.gradle-toolset

```diff
@@ -11,7 +11,9 @@ ARG GRADLE_WRAPPER_DOWNLOAD_SHA256=29e49b10984e585d8118b7d0bc452f944e386458df273
 
 USER root
 
-RUN microdnf install --nodocs openssl-${OPENSSL_VERSION}* && microdnf clean all
+RUN microdnf install --nodocs openssl-${OPENSSL_VERSION}* && \
+    microdnf install --nodocs git && \
```

Comment:
Could you align this with https://github.com/opendevstack/ods-pipeline/blob/master/build/package/Dockerfile.start#L49? That way we have the git version used everywhere.